content/pytorch/concepts/tensors/terms/tensor/view/view.md

```diff
@@ -0,0 +1,162 @@
+---
+Title: '.view()'
+Description: 'Returns a new tensor with the same data as the self tensor but of a different shape or type'
+Subjects:
+  - 'Data Science'
+  - 'Machine Learning'
+Tags:
+  - 'AI'
+  - 'Data Types'
+  - 'Deep Learning'
+  - 'Functions'
+  - 'Math operations'
+  - 'Data Science'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'py-torch-for-classification'
+---
+--------------
+
+# Tensor.view(*shape)
+Returns a new tensor with the same data as the self tensor but of a different shape.
+
+The **`.view()`** method in PyTorch is used to reshape the dimensions of a tensor without changing the underlying data. The `.view()` method returns a new tensor with the same number of elements as the original tensor but with a different shape. The `.view()` method for a Tensor object in PyTorch takes the arguments (subspaces), for example, r and c, to reshape the tensor into a matrix with r rows and c columns. The first argument specifies the desired number of rows and the second argument specifies the desired number of columns. The `.view()` method takes in one or more integers as arguments, which specify the desired shape of the tensor. The arguments can be a single integer, in which case the tensor is reshaped into a 1−dimensional tensor with the specified length. Setting -1 as the 1st number can adjust the size automatically so you don't need to set 12, 2, 3, 4 or 6 as the 1st number. *-1 is available only as the 1st number. Alternatively, the arguments can be a tuple of integers, in which case the tensor is reshaped into a tensor with the specified shape.
+
+## Syntax
+
+shape (torch.Size or int...) – the desired size
```

Comment:
The syntax should be wrapped in the ```pseudo``` block. And the syntax for this is:

```pseudo
tensor.view(*shape)
```

Below this syntax please describe the `shape` parameter as follows:

-`*shape`: <its description>